detox/ios/Detox/Utilities/UIView+DetoxUtils.m

```diff
@@ -277,14 +277,11 @@ - (BOOL)_dtx_testVisibilityInRect:(CGRect)rect percent:(NSUInteger)percent
 	CGRect testedRegionInWindowCoords = [windowToUse convertRect:rect fromView:self];
 	
 	CGRect visibleBounds = self.dtx_visibleBounds;
-	
-	if (CGRectIsNull(visibleBounds) || [self _dtx_isRegionObscured:visibleBounds
-		 										  fromTestedRegion:self.dtx_visibleBounds
-														   percent:percent]) {
+	if (CGRectIsNull(visibleBounds) || CGRectIsEmpty(visibleBounds) ||
```

Comment:
Apple say [`GCRecIsEmpty`](https://developer.apple.com/documentation/coregraphics/1454917-cgrectisempty) checks for nullability. Does that render `GCRectIsNull` redundant?

Reply:
definitely right.